index.js

```diff
@@ -97,6 +97,13 @@ function fastifyWebsocket (fastify, opts, next) {
     const response = new ServerResponse(request)
     request[kWs] = WebSocket.createWebSocketStream(connection)
     request[kWs].socket = connection
+
+    request[kWs].socket.on('newListener', event => {
+      if (event === 'message') {
+        request[kWs].resume()
+      }
+    })
```

Comment:
I think you can remove https://github.com/fastify/fastify-websocket/pull/81/files#diff-168726dbe96b3ce427e7fedce31bb0bcR83-R87 now.

Reply:
Done